css/sections.css

```diff
@@ -28,6 +28,18 @@ section {
   border: 3px solid #fff;
 }
 
+.very-long-title {
```

Comment:
use classes in BEM style: `.sectionTitle--md` , `.sectionTitle--lg`, `.sectionTitle--xl`

Reply:
it's simple and makes sense